x/ecocredit/core/genesis_test.go

```diff
@@ -535,7 +388,17 @@ func TestGenesisValidate(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.id, func(t *testing.T) {
-			err := core.ValidateGenesis(tc.genesisState(), tc.params)
+			ormCtx := ormtable.WrapContextDefault(ormtest.NewMemoryBackend())
+			modDB, err := ormdb.NewModuleDB(&ecocredit.ModuleSchema, ormdb.ModuleDBOptions{})
+			require.NoError(t, err)
+			ss, err := api.NewStateStore(modDB)
+			require.NoError(t, err)
+			tc.setupState(ormCtx, ss)
+			target := ormjson.NewRawMessageTarget()
+			require.NoError(t, modDB.ExportJSON(ormCtx, target))
+			jsn, err := target.JSON()
+			require.NoError(t, err)
+			err = core.ValidateGenesis(jsn, tc.params)
```

Comment:
This was much cleaner with the raw json and I think we want to test raw json for genesis.

Reply:
i can reduce this to a utility function to improve readability. 

> test raw json for genesis 

not sure i see the benefit of this

Reply:
> i can reduce this to a utility function to improve readability.

Not necessary if it's not being reused but ok either way.